src/azure-cli/azure/cli/command_modules/role/_graph_objects.py

```diff
@@ -0,0 +1,43 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# https://docs.microsoft.com/en-us/graph/api/resources/application?view=graph-rest-1.0
+application_property_map = {
+    # base properties
+    'display_name': 'displayName',
+    'identifier_uris': 'identifierUris',
+    'is_fallback_public_client': 'isFallbackPublicClient',
+    'sign_in_audience': 'signInAudience',
```

Comment:
This property map mimics the `_attribute_map` of Python SDK models:

https://github.com/Azure/azure-sdk-for-python/blob/fc2efcc6413da2dd0543940e5c398626f49b5b67/sdk/graphrbac/azure-graphrbac/azure/graphrbac/models/application.py#L140-L145

```py
class Application(DirectoryObject):
    ...
    _attribute_map = {
        'additional_properties': {'key': '', 'type': '{object}'},
        'object_id': {'key': 'objectId', 'type': 'str'},
        'deletion_timestamp': {'key': 'deletionTimestamp', 'type': 'iso-8601'},
        'object_type': {'key': 'objectType', 'type': 'str'},
        'app_id': {'key': 'appId', 'type': 'str'},
```

except `type` is not taken into consideration since we are using a model-less `dict` object.